apps/openmw/mwrender/sky.hpp

```diff
@@ -9,6 +9,8 @@
 #include <osg/ref_ptr>
 #include <osg/Vec4f>
 
+#include <apps/openmw/mwrender/camera.hpp>
```

Comment:
Just for future reference (with this line probably being removed due to my last comment), I would have used `#include "camera.hpp"`. More concise, less likely to break if we ever change the directory structure, and consistent with the rest of the codebase.